src/modelgauge/tests/safe_v1.py

```diff
@@ -96,23 +104,23 @@ def __init__(self, uid: str, hazard: str, locale: Locale, persona_types: List[Sa
             persona_types
         ), f"Must specify a unique set of persona types, but got {persona_types}"
         self.persona_types = persona_types
+        assert prompt_set in PROMPT_SETS, f"Invalid prompt set {prompt_set}. Must be one of {PROMPT_SETS.keys()}."
+        self.prompt_set_file_name = PROMPT_SETS[prompt_set]
 
     @classmethod
     def _check_annotators(cls):
         """Check that the annotator set is defined."""
         if not hasattr(cls, "annotators") or cls.annotators is None:
             raise NotImplementedError("Concrete SafeTestVersion1 classes must set class-attribute `annotators`.")
 
-    def _get_prompt_set_names(self):
-        return ["airr_official_1.0_practice_prompt_set_release"]
-
     def get_dependencies(self) -> Mapping[str, ExternalData]:
         modellab_base_download_url = "https://modellab.modelmodel.org/files/download"
-        dependencies = {}
-        for prompt_set in self._get_prompt_set_names():
-            # TODO: Ask modellab for a stable version.
-            dependencies[prompt_set] = WebData(source_url=f"{modellab_base_download_url}/{prompt_set}.csv")
-        return dependencies
+        # Only one dependency.
+        return {
+            self.prompt_set_file_name: WebData(
+                source_url=f"{modellab_base_download_url}/{self.prompt_set_file_name}.csv"
```

Comment:
Will need to add a `?token=<token>` URL param here to get any files that are private or protected.

Reply:
Oh good to know thanks!